client/index.html

```diff
@@ -1,11 +1,116 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <template data-type="pug" data-src="./view/base/meta.pug" />
-    <link rel="stylesheet" href="./index.css">
-    <script type="module" src="./index.js" defer></script>
-  </head>
-  <body>
-    <template data-type="pug" data-src="./view/browsers/browsers.pug" />
-  </body>
+<head>
+  <link rel="stylesheet" href="./index.css">
+  <script type="module" src="./index.js" defer></script>
+  <meta name="color-scheme" content="dark light">
+</head>
+
+<body>
+  <article class="Article Article--left">
+    <section class="Introduction">
+      <div class="Introduction__header">
+        <h1 class="Article__h1">
+          <img src="./assets/browserlist_logo.svg" role="presentation">
+          Browserslist
+        </h1>
+        <a class="Badge" href="https://github.com/browserslist/browserslist">
+          <img src="./assets/github_logo.svg" alt="">
+          10.5k
+        </a>
+        <a class="Badge" href="https://twitter.com/Browserslist">
+          <img src="./assets/twitter_logo.svg" alt="">
```

Comment:
We need `alt` here. Otherwise it will be just `4k` in screen reader.

Reply:
There must be a `presentation`, not alt. I am going to add)

Reply:
In this case, we need some text for screen readers by other way